packages/api/src/util/mockAddonPartner.ts

```diff
@@ -39,11 +39,11 @@ export const getIrrigationPrescriptions = async ({
 }: fakeIrrigationPrescriptionsProps): Promise<ExternalIrrigationPrescription[]> => {
   const PRESCRIPTION_CONFIG = [
     {
-      id: 1,
+      id: new Date(startTime ?? Date.now()).getUTCDate(),
```

Comment:
Since this only updates one new one per day, it might still be worth showing Denis the correct column in `irrigation_task` table in beta db to delete the id. He should have access to that.